src/components/AdminSettings.vue

```diff
@@ -95,6 +104,18 @@ export default {
 				return
 			}
 			OCP.AppConfig.setValue('ldap_write_support', 'template.user', this.userTemplate)
+			},
+			setGroupTemplate() {
+				if(this.templates.group === "") {
+					let self = this;
+					OCP.AppConfig.deleteKey('ldap_write_support', 'template.group', {
+						success: function() {
+							self.templates.group = self.templates.groupDefault;
+						}
+					});
+					return;
+				}
+				OCP.AppConfig.setValue('ldap_write_support', 'template.group', this.templates.group);
```

Comment:
Should be `this.groupTemplate` to be consistent, or change the user template to `this.templates.user`.